room_access_rules/__init__.py

```diff
@@ -346,9 +346,9 @@ async def check_event_allowed(
                 State events in the room the event originated from.
 
         Returns:
-            True if the event should be allowed, False if it should be rejected,
-            or a dictionary if the event needs to be rebuilt (containing the event's
-            new content).
+            True if the event should be allowed, False if it should be rejected. Also
+            always returns None as a second parameter since we don't replace event
+            contents here.
```

Comment:
I think we shouldn't say "returns true if..." when we're returning a tuple. How about:

```suggestion
            A 2-tuple (allowed, None). `allowed` is True if the event should be
            allowed, False if it should be rejected. The second entry is always
            None because this module doesn't replace event contents.
```